coding/cpp_style_guide.rst

```diff
@@ -2079,87 +2087,88 @@ Example 2 adds an additional line without significant increase in readability.
 
 .. code-block:: cpp
 
-   class SomeClass :
-       public  BaseClass1,
-       public  BaseClass2,
-       private BaseClass3
-   {
+   class SomeClass : public BaseClass1, public BaseClass2, private BaseClass3 {
    public:
        SomeClass() {}
+   
    protected:
        ...
+   
    private:
        ...
    };
 
 Note that:
 
-- Like functions, the opening brace is at the same indentation as the class keyword
-- any subclasses are indented one block and their visibility is explicitly qualified
-- The declarations public, protected and private are left-justified to the same indentation as class.
-- If ``SomeClass`` is not derived from anything, the opening brace should still line up with class
-
-For historical reasons, the format:
-
-.. code-block:: cpp
-
-   class SomeClass : public BaseClass1, BaseClass2, private BaseClass3 {
-   public:
-       SomeClass() {}
-   protected:
-       ...
-   private:
-       ...
-   };
-
-is permitted but not encouraged, and should not be used in new code.
+  - Any base class name should be on the same line as the subclass name,
+    subject to the 110-column limit.
+  - The ``public:``, ``protected:``, and ``private:`` keywords should not be indented.
+  - Except for the first instance, these keywords should be preceded by a blank line.
+    This rule is optional in small classes.
+  - Do not leave a blank line after these keywords.
+  - The ``public`` section should be first, followed by the ``protected`` and finally the ``private`` section.
 
 .. _style-guide-cpp-6-6:
 
 6-6. Function declarations MAY have any of the following three forms:
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Return type on the same line as function name, parameters on the same line if they fit.
```

Comment:
Note when publicizing that there are changes in this area, particularly for opening brace.